include/solvers/gls_nitsche_navier_stokes.h

```diff
@@ -46,6 +46,9 @@ class GLSNitscheNavierStokesSolver : public GLSNavierStokesSolver<spacedim>
                                const unsigned int degreeVelocity,
                                const unsigned int degreePressure);
 
+  virtual void
+  solve() override;
```

Comment:
Very good habit. Always override virtual functions, this way you are sure of what you are overriding